atd-vzv/src/views/summary/CrashesByMode.js

```diff
@@ -339,7 +358,7 @@ const CrashesByMode = () => {
                                   {chart.data.datasets.map(
                                     (mode, modeIterator) => {
                                       let paddingBottom =
-                                        modeIterator === 4 ? "pb-1" : "pb-0";
+                                        modeIterator === 4 ? "pb-0" : "pb-0";
```

Comment:
To avoid further confusion down the road with the meaning of this magic number 4, could you remove this `paddingBottom` var altogether? 🧹 Thanks again for getting these totals in here - it really ties this viz together 🙏

Reply:
yeah i definitely did not mean to leave this in here, was gonna continue working on yalls requests today after doing a quick commit last night :facepalm:

Reply:
oh gotcha - just me being eager here! 😅